testsuite/pyproject.toml

```diff
@@ -72,6 +72,7 @@ include=[
     "MDAnalysisTests.data.gromacs",
     "MDAnalysisTests.data.lammps",
     "MDAnalysisTests.data.mol2",
+    "MDAnalysisTests.data.dssp",
```

Comment:
This specific folder location is missing an `__init__.py` file - could you add one please?